src/Profilers/MonitorProfiler/Communication/ManagedMessageCallbackManager.h

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include <unordered_map>
+#include <mutex>
+#include "cor.h"
+#include "corprof.h"
+#include "Messages.h"
+
+typedef INT32 (STDMETHODCALLTYPE *ManagedMessageCallback)(UINT16, const BYTE*, UINT64);
```

Comment:
```suggestion
typedef HRESULT (STDMETHODCALLTYPE *ManagedMessageCallback)(UINT16, const BYTE*, UINT64);
```
nit: I know this returned a `INT32` prior to this change, but it probably should be `HRESULT`. I know when it all gets resolved it ends up basically being the same, but all of the implementers are return `HRESULT` and the caller is returning it as an `HRESULT`.